UPGRADING.md

```diff
@@ -1,3 +1,15 @@
+## From v3 to v4
+### Model event Logging
+
+- All models now need to define a `getActivitylogOptions()` method to configure and return the models options as a `LogOptions` instance.
+- To control what attributes are logged, instead of defining a `$logAttributes` property this is defined in the `getActivitylogOptions()` method using the `logOnly()` method of `LogOptions`.
+- The `getDescriptionForEvent()` method is no longer used to customize the description. Instead, use the `setDescriptionForEvent()` method for `LogOptions` class. 
+- When customizing the log's name instead of defining a `$logName` property, just call the `useLogName()` method when configuring the `LogOptions`.
```

Comment:
```suggestion
- When customizing the log's name instead of defining a `$logName` property, call the `useLogName()` method when configuring the `LogOptions`.
```